src/LodeRunner.API/src/Program.cs

```diff
@@ -117,6 +117,12 @@ public static async Task<int> RunApp(Config config)
                 // end app on error
                 GetLogger().LogError(new EventId((int)HttpStatusCode.InternalServerError, nameof(RunApp)), ex, "Exception");
 
+                // log inner exception
+                if (ex.InnerException != null)
+                {
+                    GetLogger().LogError(new EventId((int)HttpStatusCode.InternalServerError, nameof(RunApp)), ex.InnerException, "InnerException");
+                }
```

Comment:
This will create two entries for the same exception.  Let's check for the inner exception and log a combined message of inner and outer exception in a single entry.  There should be a way to have it log whatever exists.  Check the `LogError` method as it is probably logging just the top-level exception, but we should have the `LogError` method handle logging the full exception and not have to put special code in different places.